platform/cog-platform-fdo.c

```diff
@@ -1069,7 +1026,26 @@ on_export_buffer_resource (void* data, struct wl_resource* buffer_resource)
                                                    image_attrs);
     assert (wpe_view_data.image != NULL);
 
-    draw ();
+    static PFNEGLCREATEWAYLANDBUFFERFROMIMAGEWL
+        eglCreateWaylandBufferFromImageWL;
+    if (eglCreateWaylandBufferFromImageWL == NULL) {
+        eglCreateWaylandBufferFromImageWL =
+            eglGetProcAddress ("eglCreateWaylandBufferFromImageWL");
+        assert (eglCreateWaylandBufferFromImageWL != NULL);
```

Comment:
Please replace the assertion with:

```c
g_assert_nonnull (eglCreateWaylandBufferFromImageWL);
```

Using `g_assert*` gets us better messages when the assertion checks fail.